pkg/oc/cli/admin/policy/reconcile_sccs.go

```diff
@@ -25,7 +25,6 @@ import (
 	securityv1 "github.com/openshift/api/security/v1"
 	securityv1typedclient "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
-	securityapiv1 "github.com/openshift/origin/pkg/security/apis/security/v1"
 )
 
 // ReconcileSCCRecommendedName is the recommended command name
```

Comment:
you should get permission to remove this entire command.

Reply:
@ericavonb @enj WDYT? ^^